error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -377,6 +377,10 @@ Flux<Integer> testFluxFilterSortWithComparator() {
     return Flux.just(1, 4, 3, 2).sort(reverseOrder()).filter(i -> i % 2 == 0);
   }
 
+  Mono<ImmutableSet<Integer>> testFluxCollectToImmutableSet() {
+    return Flux.just(1, 4, 3, 2).collect(toImmutableList()).map(ImmutableSet::copyOf);
```

Comment:
On a second thought, let's simplify the numbers here. I think it makes sense to have `1, 2` as we are collecting and "usually" there will be multiple values to collect. The order used in the test above is important to the method that is used.

Reply:
Should we also simplify for https://github.com/PicnicSupermarket/error-prone-support/pull/570?

Reply:
Yes, will push a commit :).